### ISSUE TITLE
xcresult 1.18 fails with an exception

### ISSUE THREAD

**Issue by cheshi-mantu** [eroshenkoam/xcresults#72]
Вызываю:
./xcresults export "<MASKED>Tests-E2EDebug-116-1.xcresult" allure-results

Получаю:
Export xcresults from [<MASKED>Tests-E2EDebug-116-1.xcresult] to [allure-results]
Export information about 2 test summaries...
Jul 17, 2024 11:07:01 AM com.fasterxml.jackson.databind.ext.Java7Handlers
WARNING: Unable to load JDK7 types (java.nio.file.Path): no Java7 type support added
java.lang.NullPointerException
at io.eroshenkoam.xcresults.export.Allure2ExportFormatter.getPosition(Allure2ExportFormatter.java:270)
at io.eroshenkoam.xcresults.export.Allure2ExportFormatter.format(Allure2ExportFormatter.java:109)
at io.eroshenkoam.xcresults.export.ExportProcessor.export(ExportProcessor.java:127)
at io.eroshenkoam.xcresults.export.ExportCommand.runUnsafe(ExportCommand.java:74)
at io.eroshenkoam.xcresults.export.ExportCommand.run(ExportCommand.java:62)
at picocli.CommandLine.executeUserObject(CommandLine.java:1769)
at picocli.CommandLine.access$900(CommandLine.java:145)
at picocli.CommandLine$RunLast.handle(CommandLine.java:2141)
at picocli.CommandLine$RunLast.handle(CommandLine.java:2108)
at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1975)
at picocli.CommandLine.execute(CommandLine.java:1904)
at io.eroshenkoam.xcresults.XCResults.main(XCResults.java:13)

**Comment by mikehouse** [comment]
@cheshi-mantu, what Xcode version did you use ?

**Comment by havebeenfitz** [comment]
@mikehouse seeing a similar error on `export` with xcresults 1.18 and Xcode 16 

```
Delete existing output directory...
Export xcresults from [DerivedData/Logs/Test/Run-App-2024.09.20_11-09-11--0600.xcresult] to [./build/reports/output/ui_tests_CI]
java.lang.NullPointerException
	at io.eroshenkoam.xcresults.export.ExportProcessor.export(ExportProcessor.java:88)
	at io.eroshenkoam.xcresults.export.ExportCommand.runUnsafe(ExportCommand.java:74)
	at io.eroshenkoam.xcresults.export.ExportCommand.run(ExportCommand.java:62)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1769)
	at picocli.CommandLine.access$900(CommandLine.java:145)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2141)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2108)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1975)
	at picocli.CommandLine.execute(CommandLine.java:1904)
	at io.eroshenkoam.xcresults.XCResults.main(XCResults.java:13)
```

**Comment by eroshenkoam** [comment]
Hey everyone! 🙏

Apologies for the delay in resolving this issue — the past couple of weeks have been quite busy. But I’m happy to say that we’ve finally addressed it in the [1.19.0](https://github.com/eroshenkoam/xcresults/releases/tag/1.19.0) release! 🚀

Big thanks to everyone who contributed and provided feedback in this ticket.

Let’s keep pushing forward! 💪
